modules/standard/IO.chpl

```diff
@@ -1337,7 +1336,7 @@ proc file.unlock() {
 // channel style is protected by channel lock, can be modified.
 pragma "no doc"
 proc file._style:iostyle {
-  check();
+  try! check();
```

Comment:
Confused. Why is using `try!` here better than just halting in `file.check` ? Isn't the point here to remove halts from the standard library?

Reply:
I have no idea what I was thinking. Corrected.